include/modem/modem_info.h

```diff
@@ -11,6 +11,8 @@
 #include <cJSON.h>
 #endif
 
+#include <at_cmd_parser/at_params.h>
```

Comment:
Has been moved to `#include <modem/at_params.h>`

Reply:
Monday, right? :) I fix.